### ISSUE TITLE
Replace color of tab text to have a contrast of 4.5 or higher

### ISSUE THREAD

**Issue by paul-rinaldi** [codeformilwaukee/DECARCERATION-PLATFORM#75]
#17543c

"Just some very low hanging fruit on the subject of accessibility, I think the contrast for the links could definitely use some improvement. Color #17543c would fulfill WCAG standards of 4.5.

As a sidenote for accessibility, 
https://www.bjs.gov/index.cfm?ty=pbdetail&iid=5500#:~:text=Highlights%3A,having%20at%20least%20one%20disability.

Inmates are generally 3 times as likely to report having a disability, so I would argue that accessible design should be at the forefront of this."

**Comment by KyleWestendorf** [comment]
Grabbing this.
